test/e2e/integration/signin.spec.js

```diff
@@ -0,0 +1,107 @@
+import { REGISTER_API, LOGIN_API, ME_API } from '../apiRoutes'
+describe('SignIn test page', () => {
+  beforeEach(() => {
+    cy.visit('/signin')
+
+    cy.server()
+  })
+
+  it('Should register and redirect to home', () => {
+    cy.get('#signInFirstName').type('John')
+    cy.get('#signInLastName').type('Doe')
+    cy.get('#signInEmail').type('john@doe.com')
+    cy.get('#signInPassword').type('jDoe')
+    cy.get('#signInPasswordConfirm').type('jDoe')
+
+    cy.fixture('account.json').then((data) => {
+      cy.route('POST', REGISTER_API, data.register.success, { status: 200 }).as('registerSuccess')
+    })
+    cy.fixture('account.json').then((data) => {
+      cy.route('POST', LOGIN_API, data.login.success, { status: 200 }).as('loginSuccess')
+    })
+    cy.fixture('account.json').then((data) => {
+      cy.route('GET', ME_API, data.me.success, { status: 200 }).as('meSuccess')
+    })
+    
+    cy.get('#signInButton').click()
+    cy.wait('@registerSuccess')
+    cy.wait('@loginSuccess')
+    cy.wait('@meSuccess')
+
+    cy.url().should('include', 'home')
+  })
+
+  it('Should show error if the FirstName field is empty', () => {
+    cy.get('#signInFirstName').type('John')
+    cy.get('#signInFirstName').clear()
+    cy.get('#signInFirstName-live-feedback').should('be.visible')
+  })
+
+  it('Should show error if the LastName field is empty', () => {
+    cy.get('#signInLastName').type('Doe')
+    cy.get('#signInLastName').clear()
+
+    cy.get('#signInLastName-live-feedback').should('be.visible')
+  })
+
+  it('Should show error if the mail field isn\'t correctly filled', () => {
```

Comment:
You can use this symbol  `‘` (`alt` + `4`) on a mac. 
```suggestion
it('Should show error if the mail field isn‘t correctly filled', () => {
```